Makefile

```diff
@@ -1,536 +0,0 @@
-# Copyright 2016 The Kubernetes Authors.
```

Comment:
So developers can still run `make foo` in kubernetes root directory, right? I'm not sure if there is something path-related in original Makefile.